website/docs/concepts/event-bridge/index.md

```diff
@@ -28,6 +26,262 @@ To jump right into building, see the [`EventBus`](./event-bus.md) documentation.
 
 An instance of an [`EventBus`](./event-bus.md) ingest events and routes them to downstream integrations according to Rules created by the user. Events are sent to downstream services such as Lambda Functions, Step Functions, or a third party (non-AWS) API. Sources of events include other AWS Resources or third party (non-AWS) SaaS products, e.g. a Slack webhook.
 
-## Default Event Bus
+## Declare an Event Type
+
+Functionless supports well typed events, lets add our event schema to Typescript.
+
+```ts
+interface UserDetails {
+  id?: string;
+  name: string;
+  age: number;
+  interests: string[];
+}
+
+interface UserEvent
+  extends functionless.EventBusEvent<
+    UserDetails,
+    // We can provide custom detail-types to match on
+    "Create" | "Update" | "Delete"
+  > {}
+```
+
+## Create an Rule
```

Comment:
`Add a`

Reply:
Find replace needs to support grammar updates.